backend/app/app/db/connection.py

```diff
@@ -0,0 +1,22 @@
+import os
+import psycopg2
+
+POSTGRES_USER = os.environ['POSTGRES_USER']
+POSTGRES_PASSWORD = os.environ['POSTGRES_PASSWORD']
+POSTGRES_DB = os.environ['POSTGRES_DB']
+
+
+def create_connection():
```

Comment:
Connection credentials are present in `core.config`

Reply:
Thanks. Updated the code with usage of `core.config` instead.